src/quickstart/provider/vagrant/boot.md

```diff
@@ -27,11 +27,10 @@ See [Kubernetes Versions](https://deis.com/docs/workflow/installing-workflow/sys
 This archive has everything that you need to launch Kubernetes. It's a fairly large archive, so it may take some time to download:
 
 ```
-$ curl -sSL https://storage.googleapis.com/kubernetes-release/release/v1.3.5/kubernetes.tar.gz -O
-$ tar -xvzf kubernetes.tar.gz
+$ curl -sSL https://storage.googleapis.com/kubernetes-release/release/v1.5.1/kubernetes.tar.gz | tar xzv
```

Comment:
For example, there are techniques programmers can use to distinguish whether or not you're piping to shell or not and they can send different things to you if you are.